docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md

```diff
@@ -39,26 +41,24 @@ If the amount of data on your disk is much less than the size of the disk, then
 
 ### Copy and Download the Disk
 
-1.  Copy the disk over SSH from the Linode to the receiving machine. Replace `192.0.2.9` with the Linode's IP address and `/home/archive/linode.img` with the path where you want to store the disk.
+1.  Prepare the Receiving System that will be downloading the disk image by verifying that SSH is installed. Most Linux/Unix-like systems include OpenSSH in their package base by default.
```

Comment:
* Receiving System doesn't need to be capitalized 
* Please remove future tense.